app/static/js/slcNewProposal.js

```diff
@@ -78,6 +78,18 @@ $(document).ready(function(e) {
   });
 
   $("#cancelButton").on("click", function() {
+    $.ajax({
+      url: '/serviceLearning/canceledProposal',
+      method: 'POST',
+      data: {courseID : document.getElementById('courseID').value},
+      success: function(response) {
+          msgToast("Proposal Canceled", "Successfully canceled the proposal edit/creation.")
+      },
+      error: function(error) {
+          msgFlash('Error: ', error)
```

Comment:
Is there a reason we are using flash on error and toast on success? we probably want to be consistent.

Reply:
I will can change both to msgFlash, but is there a type that you would prefer us to be consistent with?